### DISCUSSION TITLE
Disconnect after a few days

### DISCUSSION THREAD

**Discussion by CordasMC** [PrismarineJS/mineflayer#3092] [Q&A]
I’ve written a bot that uses an autoreconnect feature to stay connected to a server, but after a couple days it just doesn’t reconnect anymore and I have to restart the KVM I’m hosting it on for it to be able to connect to the server again. I’m not that experienced regarding mc servers so I’m hoping someone can help out. Thanks in advance!

**Comment by KrisLogic** [comment]
use pm2 or other can autostart application may be eazy , don't have to learn any code language just read the tutorial
